### PR TITLE
Awesome-books-es6-modules

### PR DESCRIPTION
**Pull request by silviatofana**
Setup your project:
Create a new directory and initialize it with git.
Use the npm init -y command to create a package.json file.
Create the entry point for your JavaScript code called index.js
Create an index.css file for your styles.
Create the main index.html file. Link your index.js and index.css files there.
Create a directory called modules.
Re-create your Awesome books app following the rules:

Use ES6 syntax.
Divide your code into modules. Save each module in a separate file in the modules dir. Import modules in the index.js file. For this exercise - the more modules the better!
Keep all functionalities of the app without errors.
Refactor your methods and functions to arrow functions.
Use let and const in the correct way.
Display the current date below the navigation bar.

For this requirement, you should use a library [Luxon]
Download the script and save it to your project.
Import it as a module.
See the article [Get the current date and time]. Your task here is to use a new tool effectively with little externally curated support. Exactly as you will do in your future job.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ In this website you can add books of your choice to the list of books and save t
```
Comment (TedLivist) [inline README.md:22]: - [ ] In keeping with ES6 syntax kindly refactor your functions to be arrow functions.

  BookList.js | Book.js
  :---:|:---:
  silvia-1 | silvia-2

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @silviatofana,

Good job so far!
✔️ Good documentation
✔️ Neat UI
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] **[OPTIONAL]** It would be nice to reset the values of the input boxes after a book has been added

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (silviatofana) [reply README.md:22]: ok I will thanks

Comment (silviatofana) [reply README.md:22]: please kindly check here I already use the arrow function
image

Comment (TedLivist) [reply README.md:22]: @silviatofana yeah. You used arrow functions but it is expected that ALL your functions in your files are arrow functions.

Comment (silviatofana) [reply README.md:22]: in all module files or in all js files ?

Comment (TedLivist) [reply README.md:22]: @silviatofana all JS files that are relevant to this particular project.

Comment (TedLivist) [reply README.md:22]: Whether in the `modules` folder or outside

Comment (silviatofana) [reply README.md:22]: image

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @silviatofana👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by silviatofana** [comment]
wow thank you so much, thank God for such a good and understanding reviewer like you have Microverse

**Comment by thecodechaser** [comment]
> wow thank you so much, thank God for such a good and understanding reviewer like you have Microverse

@silviatofana Your code passes all requirements for approval. (You should avoid using arrow functions in class read more [here](https://stackoverflow.com/questions/31362292/how-to-use-arrow-functions-public-class-fields-as-class-methods)).

**Pull request merged**